en/gps_compass/rtk_gps_locosys_r1.md

```diff
@@ -1,21 +1,23 @@
-# Locosys RTK GPS
+# LOCOSYS Hawk R1 RTK GPS

```

Comment:
The titles need to be differentiated now we have two of them

Reply:
Yes, Thanks, had to.